server/server.ts

```diff
@@ -1,41 +1,14 @@
-// server and backend running in Deno
-
 import { Application } from 'https://deno.land/x/oak@v10.2.0/mod.ts';
-import { connect } from "https://deno.land/x/redis/mod.ts";
-
-//imports needed to instantiate Denostore
-import { Denostore } from 'https://cdn.jsdelivr.net/gh/oslabs-beta/DenoStore/mod.ts';
-import { typeDefs, resolvers } from './schema.ts';
 
+import router from './routes/router.ts';
+import { denostore } from './routes/denostore.ts';
 
-import router from './router.ts';
+const port: number = Number(Deno.env.get('PORT')) || 3000;
```

Comment:
With a .env being used and the .env listed in gitignore we should all add a .env file with 'PORT'=3000 now right? Is that exactly what Deno wants in the .env, or is it better to have PORT=3000 without the quotes?

Reply:
Yes everyone can add that if they want to, but it is not necessary. If env value is not there, it will default to 3000. 
Also, if you do decide to add an env variable to .env file, it should look like PORT=3000 without the quotes.

Reply:
Great.